packages/constants/src/event/event.types.ts

```diff
@@ -4,7 +4,8 @@ import {
   PaymentRequestPayload,
   TrustlineRequestPayload,
```

Comment:
I would rename that type: `SetTrustlineRequestPayload` as we will have other payloads for trustlines in the future.
Same comment on line 8:
`SetTrustlineRequestPayloadDeprecated`